File: trunk/v1.0.0/src/com/youthor/bsp/core/abdf/model/Item.java
package com.youthor.bsp.core.abdf.model;

import java.io.Serializable;
import java.util.HashMap;
import java.util.Map;

public class Item implements Serializable{

    private static final long serialVersionUID = -3646449006370113095L;
    private String key;
    private String value;
    private String isUse;
    private Map extendPara=new HashMap();
    public String getKey() {
        return key;
    }
    public void setKey(String key) {
        this.key = key;
    }
    public String getValue() {
        return value;
    }
    public void setValue(String value) {
        this.value = value;
    }
    public Map getExtendPara() {
        return extendPara;
    }
    public void setExtendPara(Map extendPara) {
        this.extendPara = extendPara;
    }
    public String getIsUse() {
        return isUse;
    }
    public void setIsUse(String isUse) {
        this.isUse = isUse;
    }
    

}

File: trunk/v1.0.0/WebContent/WEB-INF/bsp/abdf/jsp/frame/51tech/js/Item.js
/*----------------------------------------------------------------------------\
|                                值和key JS
|-----------------------------------------------------------------------------
|                             作者:刘正仁
\----------------------------------------------------------------------------*/
function Item()
{
   this.key=""; 
   this.value="";

}

Item.prototype.setKey=function (key)
{
  this.key=key;
}

Item.prototype.getKey=function ()
{
   return this.key;
}

Item.prototype.setValue=function (value)
{
  this.value=value;
}

Item.prototype.getValue=function ()
{
   return this.value;
}

